final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +70,83 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
-
+scary_monster = {
+  name: "Jake from statefarm",
+  smell: "Crunchy",
+  weight: 225,
+  citiesDestroyed: ["Tokyo", "San Francisco", "Seattle", "Honolulu"],
+  luckyNumbers: [2, 24, 32, 37],
+  address: {
+    number: 1745,
+    street: "Pacific Ocean Lane",
+    state: "Pacific Ocean",
+    zip: "00000"
+  }
+}
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+class SuperHero
+  attr_accessor  :name, :super_power, :age
```

Comment:
You won't need the `attr_accessor` - will dig into what it is/difference between reader in Mod 1 classes!